.travis.yml

```diff
@@ -14,6 +14,10 @@ matrix:
             - libncurses5-dev
             - libpcre2-dev
             - python
```

Comment:
Drop `python` alone here if depending on Python 3, though I think they'll both be installed anyway.